platform/feature-api/api/src/pod-api/uri.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Creates a random UUID string with a random hexadecimal value for each character in the string
+ * 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx', and returns the result.
+ * @returns a string in UUID format
+ */
+export function createUUID(): string {
```

Comment:
would be useful to have some unit tests alongside for this as we don't depend on a lib...

e.g test that format is correct, that subsequent UUIDs are different ...

Reply:
I don't think that's really required, and it was certainly not in the original place (that had no tests at all). I can add them, no big deal.

Reply:
It's tested, it does have a test now. I can add the other tests if you want. If we have code that does the job, I really don't see a good reason to add a dependency we're not in control of.

Reply:
well, it's extra work with no extra gain... we're speaking about a universally used lib here which uses `crypto` for the randomization instead of the error prone `Math.random` we've here

Reply:
Irini, I really don't think we need a cryptographically secure ID generated here. Use case here is to have at most a dozen imports with their corresponding UUIDs generated. We don't even need to have an UUID, it was only chosen for convenienze. We could have called it polyID and we would be none the wiser. Boils down to, our use case does not require another library thrown in here. If you really think we need to check for non-repeating UUIDs as many times as the expected use, fine, I can do that. Format is already checked in the (new) unit tests, please check that out. But really, no new dependencies if they are not totally, absolutely, needed.

Reply:
Actually, it was @shoniko in #186 who introduced it. Also, `uuid` is indeed installed at the top level (as a dependency of second degree). If it's going to be a blocker to get this PR accepted, I can use it, no big deal.

Reply:
I've added a test for inequality between two successive generations of UUID. Could add a test that compares one with all generated so far, but I don't think that's really needed. It's only math, really.

Reply:
... and that test told me I *really* have to use UUID. Thanks for the suggestion, it should work now.